src/objects/core/zcl_abapgit_filename_logic.clas.abap

```diff
@@ -121,8 +127,15 @@ CLASS zcl_abapgit_filename_logic IMPLEMENTATION.
     ENDIF.
 
     " handle namespaces
-    REPLACE ALL OCCURRENCES OF '/' IN rv_filename WITH '#'.
-    TRANSLATE rv_filename TO LOWER CASE.
-
+    IF is_item-obj_type = to_upper( c_json_file-extension ).
```

Comment:
For all JSON-based files like CHKO, CHKC, CHKV, EVTB this is fine.

How would we handle this (e.g., for INTF), if the file extension is `.abap`? In this case we would also need a different replacement for namespaces, don't we? Can we distinguish this in this method whether an object supports the ABAP file format?

Reply:
list of object types + check experimental flag

Reply:
Mmmh, I think for CHKV, CHKO, CHKC, EVTB (which only support AFF) we need to do it without experimental flag.
For INTF (and others which still support the old format) we should check also the experimental flag.

Would you suggest to implement it this way?